pkgs/by-name/be/betterdisplaycli/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  stdenv,
+  swift,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "betterdisplaycli";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "waydabber";
+    repo = "betterdisplaycli";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rfJDoPzIsBaKkGhgsGciErAoDycMDojxjYnT+oQZjnA=";
+  };
+
+  buildInputs = [ swift ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp betterdisplaycli $out/bin
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Simple CLI access for BetterDisplay";
+    homepage = "https://github.com/waydabber/betterdisplaycli";
+    license = lib.licenses.mit;
+    mainProgram = "betterdisplaycli";
+    maintainers = [ ];
```

Comment:
The package needs a maintainer

If you are not already, add yourself to the `maintainer-list.nix` file, but do that in a separate commit, preceeding the one that adds the package with message `maintainers: add usernamegoeshere`